pkg/operator/ceph/object/user/controller.go

```diff
@@ -296,6 +296,20 @@ func (r *ReconcileObjectStoreUser) createOrUpdateCephUser(u *cephv1.CephObjectSt
 	user, err = r.objContext.AdminOpsClient.GetUser(r.opManagerContext, *r.userConfig)
 	if err != nil {
 		if errors.Is(err, admin.ErrNoSuchUser) {
+			namspacedName := types.NamespacedName{Namespace: u.Namespace, Name: "rook-ceph-object-user-" + u.Spec.Store + u.Name}
```

Comment:
We should think on the name, as it can go more than 63 characters very eaisly

Reply:
we can do something similar to how we truncate the node names in the `TruncateNodeName` method.